lib/services/pubsub/service.js

```diff
@@ -597,6 +600,7 @@ PubSubService.prototype.connectLocal = function(handler, session) {
 	var _this = this;
 
 	var socket = {
+		intraProc:true,
```

Comment:
Ahhhh, good point, I was looking at using it to check whether we need to
clone on emitting - but the check was actually more expensive than the
"half clone", so it can be removed.

On Friday, April 15, 2016, Johan <notifications@github.com> wrote:

> In lib/services/pubsub/service.js
> <https://github.com/happner/happn/pull/66#discussion_r59930687>:
>
> > @@ -597,6 +600,7 @@ PubSubService.prototype.connectLocal = function(handler, session) {
> >  	var _this = this;
> >
> >  	var socket = {
> > +		intraProc:true,
>
> Where is this variable used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/happner/happn/pull/66/files/3bf0fa902489a46e3ed9061f834ce34863b2b1f9#r59930687>
>


-- 

SIMON BISHOP

DIRECTOR

+27 (0) 83 282 0703

|     simon@tenacious.digital <brianna@tenacious.digital>

twitter: @_tenacious_ <http://www.twitter.com/_tenacious_>

|     skype: sjbishop

201 Salt Circle Arcade, Salt River, Cape Town, 7925

www.tenacious.digital